packages/builder/src/pages/builder/app/[application]/_layout.svelte

```diff
@@ -102,6 +107,15 @@
     }
   }
 
+  const showFreeTrialModal = (planType, freeTrialModal) => {
+    if (
+      planType === "enterprise_basic_trial" &&
```

Comment:
We can move this const to types and to use it here

Reply:
using plan type